pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -905,22 +962,17 @@ func (cc *XController) manageQueueJob(qj *arbv1.AppWrapper) error {
 			}
 
 			qj.Status.State = arbv1.AppWrapperStateEnqueued
-			glog.V(10).Infof("[TTime] %s, %s: WorkerBeforeEtcd", qj.Name, time.Now().Sub(qj.CreationTimestamp.Time))
-			_, err = cc.arbclients.ArbV1().AppWrappers(qj.Namespace).Update(qj)
-			if err != nil {
-				return err
-			}
-			return nil
-		}
-
-		if !qj.Status.CanRun && qj.Status.State == arbv1.AppWrapperStateEnqueued {
```

Comment:
Comment for documentation purposes. No resolution needed.

This is a performance enhancement to remove an extra round trip to etcd when preemption occurs.  In addition when round trip is removed, job is put back into the queue.